package.json

```
@@ -66,16 +66,16 @@
   "license": "Apache-2",
   "scripts": {
     "build": "gatsby build",
-    "build:production": "GATSBY_NEWRELIC_ENV=production npm run build",
-    "build:staging": "GATSBY_NEWRELIC_ENV=staging gatsby build",
-    "develop": "gatsby develop",
+    "build:production": "GATSBY_NEWRELIC_ENV=production yarn run build",
+    "build:staging": "GATSBY_NEWRELIC_ENV=staging yarn run build",
```

The `run` keyword is optional. We might want to remove it, since we don't use it in any of our other scripts:

```suggestion
    "build:production": "GATSBY_NEWRELIC_ENV=production yarn build",
    "build:staging": "GATSBY_NEWRELIC_ENV=staging yarn build",
```

---

will fix in next commit